src/steps/05-PerformanceRequirements/DOW/OtherOfferingSummary.vue

```diff
@@ -7,15 +7,18 @@
             Your {{ serviceGroupVerbiageInfo.headingSummary }}
           </h1>
           <p>
-            If you need more {{ serviceGroupVerbiageInfo.typeForText }}s, add them below. 
-            You can also edit or delete any info from the 
-            {{ serviceGroupVerbiageInfo.typeForText }}s that 
-            you have already entered. When you’re done, click "Continue" and we will 
-            move on to your 
+            If you have more requirements for this category, add them below. You can
+            also edit or delete any info from the services
+            that you have already entered. When you’re done, click “Continue” and we will
             <span v-if="nextOfferingGroupStr && !returnToDOWSummary">
-              {{ nextOfferingGroupStr }} requirements.
-            </span> 
-            <span v-else>performance requirements summary.</span>
+              move on to your {{ nextOfferingGroupStr }} requirements.
+            </span>
+            <span v-else>
+              <span v-if="showSecurityNote">
```

Comment:
I requested this conditional to be above line 13. It takes precedence over the `nextOfferingGroupStr && !returnToDOWSummary` condition. The first time through Cloud Services, if you select more than one cloud service, when leaving the offering summary page, you'll first go to the **security requirements** page, then continue onto the **next selected category**. Where you have the `v-if="showSecurityNote"` shows the incorrect message (screenshot attached).

First should be `v-if="showSecurityNotice"` then a `v-else-if` --- `v-else-if="nextOfferingGroupStr && !returnToDOWSummary"` and then finally a `v-else` with the `wrap up this category` text. Nothing should be nested in any of the `if`s

![Screen Shot 2023-04-19 at 3 29 57 PM](https://user-images.githubusercontent.com/90333349/233180729-c48bcb22-7c22-4971-9368-73333d555edf.png)